irohad/ametsuchi/impl/postgres_wsv_command.cpp

```diff
@@ -17,335 +17,379 @@
 
 #include "ametsuchi/impl/postgres_wsv_command.hpp"
 
+#include <exception>
 #include <iterator>
 #include <numeric>
 
 namespace iroha {
   namespace ametsuchi {
-
     PostgresWsvCommand::PostgresWsvCommand(pqxx::nontransaction &transaction)
         : transaction_(transaction), log_(logger::log("PostgresWsvCommand")) {}
 
-    bool PostgresWsvCommand::insertRole(const std::string &role_name) {
+    const char *kUnsuccesfulAlloc = "Unable to alloc: {}";
```

Comment:
> Is it a good decision to name variables depending on their type? I know it is a part of Google Style guide, but we do not do that in any other part of the project.

We should start doing this, it's better to have some fixed style rather than doing on thing on its own

> Also constexpr instead of const?

Kinda insignificant